packages/feedback/src/core/integration.ts

```diff
@@ -217,7 +218,7 @@ export const feedbackIntegration = (({
 
   const autoInjectActor = (): void => {
     const shadow = _createShadow(_options);
-    const actor = Actor({ buttonLabel: _options.buttonLabel, shadow });
+    const actor = Actor({ buttonLabel: (typeof _options.buttonLabel === 'object' ? (isMobile() ? _options.buttonLabel.mobile : _options.buttonLabel.desktop) : _options.buttonLabel), shadow });
```

Comment:
Can probably save bytes by putting `buttonLabel` into a const first:
```suggestion
    const buttonLabel = _options.buttonLabel;
    const actor = Actor({
      buttonLabel: (typeof buttonLabel === 'object' ? (isMobile() ? buttonLabel.mobile : buttonLabel.desktop) : buttonLabel),
      shadow
    });
```
After minification it should change from 2 instances of: `a.buttonLabel.mobile`
To be `const b = a.buttonLabel` along with the same 4 instances of which would become `b.mobile`.

Savings should be the same as if we went from 4 `a.buttonLabel.mobile` calls down to 2.